src/llmcompressor/transformers/utils/arg_parser/training_arguments.py

```diff
@@ -0,0 +1,32 @@
+from dataclasses import dataclass, field
+from typing import Optional
+
+from transformers import TrainingArguments as HFTrainingArgs
+
+__all__ = [
+    "TrainingArguments",
+]
+
+
+@dataclass
+class TrainingArguments(HFTrainingArgs):
+    """
+    Training arguments specific to LLM Compressor Transformers workflow using
+    HFTrainingArgs as base class
+
+    """
+
+    do_oneshot: Optional[bool] = field(
```

Comment:
Not specific to your PR but seems weird that we have no actual training arguments in our TrainingArugments (i.e lr, optimizer, etc)

Reply:
These are inherited from [HFTrainingArgs](https://huggingface.co/docs/transformers/v4.48.2/en/main_classes/trainer#transformers.TrainingArguments)